docs/src/catalyst_functionality/dsl_description.md

```diff
@@ -701,3 +701,66 @@ rn = @reaction_network begin
 end
 nothing # hide
 ```
+
+## Incorporating (differential) equations into reaction network models
+Some models cannot be purely described as reaction networks. E.g. consider the growth of a cell, where the rate of change in cell's volume depends on some growth factor. Here, the cell's volume would be described by a normal equation. Such equations can be incorporated into a model using the `@equations` option. Here, we create a model where a growth factor ($G$) is produced and degraded at a linear rates, and the rate of change in cell volume ($V$) os linear to the amount of growth factor:
+```@example eqs1
+using Catalyst #hide
+rn = @reaction_network begin
+  @equations begin
+    D(V) ~ G
+  end
+  (p,d), 0 <--> G
```

Comment:
Yeah, I think that makes sense. All of the volume equations are a bit awkward since I am unsure whether to go full out on modelling it properly or not.

Either way, these docs are entirely rebuilt in a follow-up PR, so let's handle that there. I mostly wrote these back in the day since I thought there was a chance this might be up for a while before I actually got to revamping the equation dosc properly.